docs/guide/rules.md

```diff
@@ -171,6 +171,7 @@ The field under validation must have the same value as the confirmation field.
 <span v-show="errors.has('password')" class="help is-danger">{{ errors.first('password') }}</span>
 
 <input v-validate="'required|confirmed:password'" name="password_confirmation" type="password" :class="{'is-danger': errors.has('password_confirmation')}" placeholder="Password, Again" data-vv-as="password">
+<span v-show="$errors.has('password_confirmation')" class="help is-danger">{{ this.$errors.first('password_confirmation') }}</span>
```

Comment:
Did you mean `errors.first` instead of `this.$errors`?

Reply:
yes, sorry I have renamed `errors` into `$errors` on my side.

Reply:
I have made the changes